src/Utils/util.test.ts

```diff
@@ -20,4 +21,201 @@ describe('util', () => {
             expect(actual).toEqual(expected)
         })
     })
+    describe('isNewActionUniqueTEXT', () => {
+        it('exercise TEXT action creation', () => {
+            const sampleAction: CLM.ActionBase = {
+                actionId: "103726c8-2cfe-46e6-ad1e-fef8dbe4d6c3",
+                actionType: CLM.ActionTypes.TEXT,
+                createdDateTime: "2019-02-25T00:00:00.000Z",
+                isTerminal: true,
+                negativeConditions: [],
+                negativeEntities: [],
+                packageCreationId: 0,
+                packageDeletionId: 0,
+                payload: '{"json":{"kind":"value","document":{"kind":"document","data":{},"nodes":[{"kind":"block","type":"line","isVoid":false,"data":{},"nodes":[{"kind":"text","leaves":[{"kind":"leaf","text":"hi","marks":[]}]}]}]}}}"',
+                requiredConditions: [],
+                requiredEntities: [],
+                requiredEntitiesFromPayload: [],
+                suggestedEntity: null,
+                version: 0
+            }
+
+            let actionSet: CLM.ActionBase[] = []
+
+            // as first action created
```

Comment:
These lines looks like descriptions of a test so just make them tests.

Reply:
Adds value by documenting the intent of the asserted tests